indico/modules/auth/client/js/signup.jsx

```diff
@@ -223,6 +258,36 @@ function Signup({
               />
             </Fieldset>
           )}
+          {termsRequireAccept && !!tos && !tosUrl && (
+            <>
+              <h3>{Translate.string('Terms of service')}</h3>
+              <div className="field policy" dangerouslySetInnerHTML={{__html: tos}} />
+            </>
+          )}
+          {termsRequireAccept && !!privacyPolicy && !privacyPolicyUrl && (
+            <>
+              <h3>{Translate.string('Privacy policy')}</h3>
+              <div className="field policy" dangerouslySetInnerHTML={{__html: privacyPolicy}} />
+            </>
+          )}
+          {termsRequireAccept && !!termsEffectiveDate && (
+            <div className="field">
+              {Translate.string('Last updated')}: {moment(termsEffectiveDate).format('LL')}
+            </div>
+          )}
+          {termsRequireAccept && (
+            <FinalCheckbox
+              name="accept_terms"
+              required
+              label={
+                <label>
+                  {Translate.string('I have read, understood, and accept the')} {termsAcceptLabel()}
```

Comment:
same problem here. this does not work for i18n -> use a single Translate string for it